ci/310-DEV_shapely20b1.yaml

```diff
@@ -0,0 +1,23 @@
+name: test
+channels:
+  - conda-forge
+  - conda-forge/label/shapely_dev
+dependencies:
+  - python=3.10
+  # required
+  - numpy
+  - pandas>=1.0
+  - pip
+  - rtree
+  - scipy>=1.0
+  - watermark
+  # testing/formatting
+  - black
+  - codecov
+  - pytest
+  - pytest-cov
+  - pytest-timeout
+  - pytest-xdist
+  # optional
+  - geopandas>=0.9.0
```

Comment:
```suggestion
  - geopandas>=0.12.0
```

Reply:
This is a final bit. Then we can merge and potentially replicate the same PR elsewhere, where we don't use pygeos directly.